src/modules/fundingManager/bondingCurve/FM_BC_BondingSurface_Redeeming_Restricted_Repayer_Seizable_v1.sol

```diff
@@ -47,42 +46,45 @@ import {SafeERC20} from "@oz/token/ERC20/utils/SafeERC20.sol";
  *              - BondingCurveBase_v1
  *              - RedeemingBondingCurveBase_v1
  *              - Repayer
- *          The contract should be used by the orchestrator admin or manager
- *          to manage all the configuration for the bonding curve as well as the
- *          opening and closing of the issuance and redeeming functionalities.
- *          The contract implements the formulaWrapper functions enforced by the
+ *          The contract should be used by the orchestrator admin to manage all
+ *          the configuration for the bonding curve as well as the opening and
+ *          closing of the issuance and redemption functionalities.
+ *          The contract implements the formulaWrapper functions enforced by
  *          using the Bonding Surface formula to calculate the issuance/
- *          redeeming rate.
+ *          redemption rate.
  *
  * @custom:security-contact security@inverter.network
  *                          In case of any concerns or findings, please refer to
  *                          our Security Policy at security.inverter.network or
  *                          email us directly!
  *
+ * @custom:version v1.0.0
+ *
+ * @custom:inverter-standard-version 0.1.0
```

Comment:
The reason why I used this version is **because** it isnt formalized yet. We still have outstanding comments in the original Document, on which I based the changes on. As this is a comment for us I would leave it until we are actually finished with version 1.0.0

Reply:
After discussion resolved